serialization/PlutusData/PlutusData.go

```diff
@@ -22,9 +22,109 @@ type _Script struct {
 	Script []byte
 }
 
-type _DatumOption struct {
-	_     struct{} `cbor:",toarray"`
-	datum []byte
+type DatumType byte
+
+const (
+	DatumTypeHash    DatumType = 0
+	DatumTypeLiteral DatumType = 1
```

Comment:
should this be Inline instead of Literal?

Reply:
Yes, I meant to change that but I forgot, thanks